doc/manual/metaprogramming.rst

```diff
@@ -118,8 +137,22 @@ instead of an :obj:`Expr`:
     julia> typeof(ans)
     Symbol
 
+:obj:`Symbol`\ s can also be created using :func:`symbol`, which takes
+a character or string as its argument:
+
+.. doctest::
+
+    julia> :foo == symbol("foo")
+    true
+
+    julia> symbol('\'')
```

Comment:
Maybe introduce `symbol("'")` first, as I found it confusing that you switch to using `'` without any apparent reason. (Is the `'` notation really useful actually?)